### PR TITLE
lunamedia: update contact info

### PR DESCRIPTION
**Pull request by bretg**

### CODE REVIEW

**Review by SyntaxNode — APPROVED** [review]

**Comment by bretg** [comment]
Actually, please hold before merging. I just tested again and this bounced.

**Comment by bretg** [comment]
ok - they fixed it.

File: static/bidder-info/lunamedia.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 endpoint: "http://rtb.lunamedia.live/?pid={{.PublisherID}}"
 maintainer:
-  email: "josh@lunamedia.io"
+  email: "cs@lunamedia.io"
```
Comment (gargcreation1992) [inline static/bidder-info/lunamedia.yaml:3]: Sent a verification email. Please reply with "received" on the email. Thanks!

Comment (gargcreation1992) [reply static/bidder-info/lunamedia.yaml:3]: @bretg Still waiting for a reply to an email.

**Pull request merged**